Header header

float32[] reflectance
float32[] distances
